adoc/vi-Physical/SA.adoc

```diff
@@ -504,7 +511,7 @@ endif::[]
 ifeval::[ "{cloud}" == "GCP" ]
 GCP
 
-Example based on `10.0.0.0/24` VPC address range. The virtual addresses must be outside of the VPC address range.
+Example based on `10.0.0.0/24` VPC address range (the virtual addresses must be outside of the VPC address range):
```

Comment:
Why you moved a very important point into brackets? Text in brackets show normally additional information and not important parts.